client/app/homePortal/homePortal.html

```diff
@@ -1,20 +1,127 @@
 <div>
-  <container id="mapContainer">
 <!--   <div class="group addressSearch">
     <label class="addressSearch">Search Address</label>
     <input class="address" type="text" value="address" ng-model="address">
     <span class="highlight"></span>
   </div> -->
-  <leaflet id="map"center="center" controls="controls" overlays="overlays" geojson="geojson" paths="paths" layers="layers" markers="markers" event-broadcast="events">
-  </leaflet>
+  <container id="mapContainer">
+    <leaflet id="map"center="center" controls="controls" overlays="overlays" geojson="geojson" paths="paths" layers="layers" markers="markers" event-broadcast="events">
+    </leaflet>
   </container>
-
-<div id="ownerDashboard">
-    <ol>
-      <li ng-click="registerAddress()"> Register Address</li>
-      <li ng-click="removeAddress()"> Remove Address</li>
-      <li ng-click="togglePermission()"> Set Permission </li>
-    </ol>
+  <div id="ownerDashboard">
+      <ol>
+        <li ng-click="showAddressForm=!showAddressForm"> Register Address</li>
+          <form ng-submit="registerAddress()" name="addressRegistry" ng-show="showAddressForm">
+            <div class="group">
+              <label>Address Line 1</label>
+              <input type='text' ng-model='newAddress.line1' required>
+              <span class="highlight"></span>
+              <span class="bar"></span>
+            </div>
+            <div class="group">
+              <label>Address Line 2</label>
+              <input type='text' ng-model='newAddress.line2'>
+              <span class="highlight"></span>
+              <span class="bar"></span>
+            </div>
+            <div class="group">
+              <label>City</label>
+              <input type="text" ng-model='newAddress.city' required>
+              <span class="highlight"></span>
+              <span class="bar"></span>
+            </div>
+            <div class="group">
```

Comment:
consider to save the states list somewhere(probably a .js file) , and load it dynamically in the future